test/one_click_install_standard_test.rb

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+
+require "tempfile"
+
+Yast.import "OneClickInstallStandard"
+
+# note partially covered also by test/lib/clients/inst_productsources_test.rb
+describe "Yast::OneClickInstallStandard" do
+  subject { Yast::OneClickInstallStandard }
+
+  let(:file) do
+    File.expand_path("data/_openSUSE_Leap_15.0_Default.xml", __dir__)
```

Comment:
BTW did you know that Trang can infer a schema for a document?

For [that file](https://github.com/yast/yast-packager/blob/master/test/data/_openSUSE_Leap_15.0_Default.xml), here's what `trang _openSUSE_Leap_15.0_Default.xml _openSUSE_Leap_15.0_Default.rnc` produces, which is useful for sanity checking of the parsing code.

```rnc
namespace os = "http://opensuse.org/Standards/One_Click_Install"

start =
  element os:metapackage {
    element os:group {
      attribute distversion { text },
      element os:repositories {
        element os:repository {
          attribute alias { xsd:NCName }?,
          attribute format { xsd:NCName },
          attribute recommended { xsd:boolean },
          element os:name {
            attribute xml:lang { xsd:NCName }?,
            text
          }+,
          element os:summary {
            attribute xml:lang { xsd:NCName }?,
            text
          }+,
          element os:description {
            attribute xml:lang { xsd:NCName }?,
            text
          }+,
          element os:url { xsd:anyURI }
        }+
      }
    }
  }
```

Reply:
interesting, we can use it now in installer to verify that xml is still same as we expect.